src/main/java/com/commercetools/emailprocessor/email/EmailProcessor.java

```diff
@@ -102,4 +108,19 @@ int callApiEndpoint(final String customObjectId, final TenantConfiguration tenan
         }
         return responseCode;
     }
+
+    String encrypt(final String value, final String encryptionKey, final int cipherMode) {
```

Comment:
maybe provide some documentation through a Javadoc or inline comments on what this method does? For me, it's quite unclear what is happening exactly on everyline.

Reply:
88e8e25